pallets/configuration/src/lib.rs

```diff
@@ -71,6 +71,14 @@ pub struct HostConfiguration {
     pub collators_per_container: u32,
     // If this value is 0 means that there is no rotation
     pub full_rotation_period: u32,
+    // Like collators_per_container but for parathreads
+    pub collators_per_parathread: u32,
+    // TODO: use a less confusing name? this is for when we support assigning 1 collator to more than 1 parathread at
```

Comment:
`parathread_assignments_per_collator`? but its longer :(

Reply:
It would be fun to call this `collator_multithreading` but that's also confusing. Anyway renaming fields shouldn't require a migration, so I am leaving this as `parathreads_per_collator` for now.